app/controllers/users.js

```diff
@@ -22,5 +24,34 @@ module.exports = {
         logger.error(`DB: ${error.errors[0]}`);
         res.status(500).json({ error: error.errors[0].message });
       });
+  },
+
+  userNewSession(req, res, next) {
```

Comment:
I just notice this, we usually use the notation:

```
exports.userNewSession = (req, res, next) => {
  ..
};
```
Instead of the way you are doing it.
Both of them are right but the one we use I think it's a little bit more flexible.